source/Use-Protocol.rst

```diff
@@ -23,15 +26,25 @@ Requests
 --------
 The requests exchange is used for round-trip information transfers in the form of T_Request messages and their resulting T_Reply messages. Routing keys for requests begin with the name of the endpoint being targeted (or broadcast, for that [special case described below](#broadcast-requests)), and may include a routing key specifier to indicate a particular command (for an OP_CMD) or attribute (for an OP_GET or OP_SET which configures an endpoint rather than assigning or querying the endpoint itself). The T_REPLY is then sent to the routing key specified in the properties provided along with the AMQP message.
```

Comment:
- case variability of T_REPLY or T_Reply (throughout)
- it feels weird to switch from talking about Request/Reply/Alert and Get/Set/Cmd over to T_Request or OP_CMD, is this standard notation? or should I have seen it somewhere already?